src/frontend/data/stores/useTimedTextTrack/index.tsx

```diff
@@ -0,0 +1,28 @@
+import create from 'zustand';
+
+import { TimedText } from '../../../types/tracks';
+import { appData } from '../../appData';
+
+interface State {
+  getTimedTextTracks: () => TimedText[];
+  timedTextTracks: {
+    [id: string]: TimedText;
+  };
+}
+
+export const [useTimedTextTrack] = create<State>((set, get) => {
+  const timedTextTracks: { [id: string]: TimedText } = {};
+
+  if (appData.video && appData.video.timed_text_tracks.length > 0) {
+    appData.video.timed_text_tracks.forEach(
+      timedTextTrack => (timedTextTracks[timedTextTrack.id] = timedTextTrack),
+    );
+  }
+
+  return {
+    getTimedTextTracks: () => {
```

Comment:
Return the timed text tracks collection instead of the object having the id as property key

Reply:
IMO the consumer can just as well use `Object.values` directly if it needs the collection instead of the indexed object.

I think those `getThing` that do not perform a get could be confusing because elsewhere we've been using `getSomething()` in those stores to fetch data from the backend.
As we're also patching methods on our stores, I'd rather stay away from methods that don't really *do* anything. What do you think?